src/main/kotlin/io/openfuture/state/domain/AddTransactionRequest.kt

```diff
@@ -0,0 +1,29 @@
+package io.openfuture.state.domain
+
+import io.openfuture.state.blockchain.dto.UnifiedTransaction
+import java.time.LocalDateTime
+import kotlin.math.pow
+
+class AddTransactionRequest(
```

Comment:
We can leave for a reason that it expects filtered transactions with wallets that are in our system. Or we can give filter responsibility to WalletService that will check whether this wallets exists and then add it

Reply:
There is no guarantee it will be filtered. You should just pass a list of unified transactions. Don't see a reason to maintain this structure. So, take this off.